tests/unit/ctl/test_transform_app.py

```diff
@@ -64,6 +65,7 @@ def test_transform_not_exist_in_infrahub_yml(tags_transform_dir: str) -> None:
             assert output.exit_code == 1
 
     @staticmethod
+    @pytest.mark.xfail(reason="Seems problematic in Python 3.9")
```

Comment:
we could use skipif here to exclude these tests in 3.9 but keep them for the other version of Python
https://docs.pytest.org/en/stable/how-to/skipping.html#id1

Reply:
Ah yes good call, updated to do this instead.